src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -278,13 +343,18 @@ static string[] MarshalUnmananagedStrArray2ManagedStrArray(IntPtr pUnmanagedStri
 		private static int Hunspell_suggest(IntPtr handle, byte[] word, out IntPtr suggestions)
 		{
 			if (MiscUtils.IsUnix)
+			{
 				return Hunspell_suggest_unix(handle, out suggestions, word);
-			return Hunspell_suggest_win(handle, word, out suggestions);
+			}
+			else
+			{
+				return Hunspell_suggest_win(handle, word, out suggestions);
```

Comment:
now that this is in an `#if __MonoCS__` block, we don't need `Hunspell_suggest_win` anymore.